tests/client/components/progressBar.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import ProgressBar from '../../../client/components/ProgressBar'
+
+// Prevent <ProgressBar> from calling the API
+ProgressBar.prototype.componentDidMount = () => {}
+
+test('test runner is working', () => {
+  expect(true).toBeTruthy()
+})
+
+test('<ProgressBar />', () => {
+  const wrapper = shallow(<ProgressBar />)
+  expect(wrapper.find('p').text()).toBe('2341')
```

Comment:
I couldn't figure out where `'2341'` should be coming from?

Reply:
Hey Bryce!
This figure ('2341') is the expected array output from mapping over the mock data